akka-testkit/src/main/scala/akka/testkit/ExplicitlyTriggeredScheduler.scala

```diff
@@ -94,9 +101,9 @@ class ExplicitlyTriggeredScheduler(@unused config: Config, log: LoggingAdapter,
       interval: Option[FiniteDuration],
       runnable: Runnable): Cancellable = {
     val firstTime = currentTime.get + initialDelay.toMillis
-    val item = Item(firstTime, interval, runnable)
+    val item = Item(UUID.randomUUID(), interval, runnable)
```

Comment:
I added uniqueness to `Item`, because otherwise, the following test would fail:
```scala
    "schedule many tasks executed once" in new TestScope {
      scheduler.scheduleOnce(1.seconds, runnable)(system.dispatcher)
      scheduler.scheduleOnce(1.seconds, runnable)(system.dispatcher)
      scheduler.timePasses(2.seconds)

      counter.get() shouldBe 2 // fails, because counter.get() is 1
    }
```